--- cl_parse.c
+++ cl_parse.c
@@ -18,5 +18,7 @@
 
 #include "quakedef.h"
 
+#define svc_qnn_training 35
+
 char *svc_strings[] =
 {
@@ -62,4 +64,5 @@
 	"svc_cdtrack",
 	"svc_sellscreen",
-	"svc_cutscene"
+	"svc_cutscene",
+	"svc_qnn_training"
 };
@@ -308,7 +311,7 @@
 // local state
 	cl_entities[0].model = cl.worldmodel = cl.model_precache[1];
 	
-	R_NewMap ();
+	QNN_ArenaNewMap ();
 
 	Hunk_Check ();		// make sure nothing is hurt
 	
@@ -769,3 +772,7 @@
+		case svc_qnn_training:
+			QNN_TrainingReadNetwork();
+			break;
+
 		case svc_nop:
 //			Con_Printf ("svc_nop\n");
 			break;
@@ -861 +868 @@
-			CL_NewTranslation (i);
+			QNN_ArenaNewTranslation (i);
